src/node/certs.h

```diff
@@ -7,4 +7,6 @@ namespace ccf
 {
   using CertDERs = kv::Map<Cert, ObjectId>;
   using CACertDERs = kv::Map<std::string, Cert>;
+  // Mapping from hex-encoded cert digest to entity id
```

Comment:
The other tables here deal with DERs, so I think we should clarify that this is actually a PEM digest:

```suggestion
  // Mapping from hex-encoded digest of PEM cert to entity id
```